doc/api/fs.markdown

```diff
@@ -989,6 +989,32 @@ The synchronous version of [`fs.writeFile()`][]. Returns `undefined`.
 
 Synchronous versions of [`fs.write()`][]. Returns the number of bytes written.
 
+## fs.mkdtemp(template, callback)
+
+Creates a unique temporary directory.
+
+The `fs.mkdtemp()` function generates a uniquely named temporary
+directory from template. The last six characters of template must be
+XXXXXX and these are replaced with a string that makes the directory
```

Comment:
Is the `XXXXXX` really necessary? Can't we just say that the temporary directory will be `template` + six additional characters and have the internals of the function handle the rest?